src/components/ProcessCreate/Census/Gitcoin/StampsUnionType.tsx

```diff
@@ -1,55 +1,82 @@
-import { Flex, FormLabel, Switch, Text } from '@chakra-ui/react'
-import { Controller, useFormContext } from 'react-hook-form'
+import { Box, Flex, HStack, Text, useRadio, useRadioGroup } from '@chakra-ui/react'
+import { useFormContext } from 'react-hook-form'
 import { useTranslation } from 'react-i18next'
-import { CensusGitcoinValues } from '~components/ProcessCreate/StepForm/CensusGitcoin'
+import { UseRadioProps } from '@chakra-ui/radio/dist/use-radio'
+import { PropsWithChildren } from 'react'
 
 export type StampsUnionTypes = 'AND' | 'OR'
 
+const UnionTypeRadioCard = (props: UseRadioProps & PropsWithChildren) => {
+  const { getInputProps, getRadioProps } = useRadio(props)
+
+  const input = getInputProps()
+  const checkbox = getRadioProps()
+
+  return (
+    <Box as='label'>
+      <input {...input} />
+      <Box
+        {...checkbox}
+        cursor='pointer'
+        borderWidth='1px'
+        borderRadius='md'
+        boxShadow='md'
+        _checked={{
+          bg: 'primary.main',
+          color: 'white',
+          borderColor: 'primary.dark2',
+        }}
```

Comment:
Sorry but this is still wrong. You should be using variables for everything, including the color set as `white`. Otherwise, if the `primary.main` color is too light, the text won't be legible. Ideally, you should use one of the variables used for buttons, since this looks like one.

Reply:
After a bit talk I implemented this:  ced3008